src/components/mini-widgets/ModeSelector.vue

```diff
@@ -16,8 +16,14 @@ datalogger.registerUsage(DatalogVariable.mode)
 const vehicleStore = useMainVehicleStore()
 const currentMode = ref()
 
-watch(currentMode, () => {
+/* eslint-disable-next-line @typescript-eslint/no-unused-vars */
+watch(currentMode, (newVal, _oldVal) => {
```

Comment:
If you just use `(newVal) => {`, will it work without having to disable the linter?

Reply:
good call. I made that change